app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split()
+    if command_split[0] == "cp" and len(command_split) == 3:
```

Comment:
first check for this `It must do nothing in case the user is trying to copy file to file with the same name`